static/bidder-params/deepintent.json

```diff
@@ -5,10 +5,10 @@
 
   "type": "object",
   "properties": {
-    "TagID": {
+    "tagId": {
       "type": "string",
       "description": "An ID which identifies the deepintent ad tag"
     }
   },
-  "required" : [ "TagID" ]
+  "required" : [ "tagId" ]
 }
```

Comment:
This is a potentially dangerous change to make as it will invalidate the configuration of any publisher currently using your adapter. Such publishers would have their entire request rejected by Prebid Server and unable to use any demand partner until they update.

This is a relative new adapter though, so perhaps it's possible no one is using it yet?

Reply:
FYI: I don't see any traffic yet for this adapter, so you're probably still in the "safe zone" unless you know of publishers using it currently.

Reply:
Yes the changes have been made to make params same as prebid js params, and as per requirement of a client itself. So its safe to merge